vignettes/estimate_infections_workflow.Rmd

```diff
@@ -230,10 +237,23 @@ def <- estimate_infections(
   delays = delay_opts(delay),
   rt = rt_opts(prior = rt_prior)
 )
-#> Warning: There were 15 divergent transitions after warmup. See
+#> Warning: There were 3 divergent transitions after warmup. See
 #> https://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
 #> to find out why this is a problem and how to eliminate them.
+#> Warning: There were 1000 transitions after warmup that exceeded the maximum treedepth. Increase max_treedepth above 15. See
```

Comment:
Problem has seemed to disappear with narrower uncertainty (based on relative variance vs. sd which I think makes sense anyway). Since max treedepth is not a huge problem anyway I think we can proceed.